### PR TITLE
Projects: remove old/non-used fields

### PR DESCRIPTION
**Pull request by humitos**
Follow safe migrations instructions from
https://dev.readthedocs.io/en/latest/migrations.html to remove these fields.

You can skip 5c556ec4bc04d3566ce88cace101f330ea48ca41 from the review since it's the lint for the `projects/models.py` file --which I'm not sure why it wasn't applied before.

### CODE REVIEW

File: readthedocs/projects/migrations/0116_remove_old_issues.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                    ("mkdocs", "Mkdocs"),
+                    ("sphinx_htmldir", "Sphinx HtmlDir"),
+                    ("sphinx_singlehtml", "Sphinx Single Page HTML"),
+                ],
+                default="sphinx",
+                help_text='Type of documentation you are building. <a href="http://www.sphinx-doc.org/en/stable/builders.html#sphinx.builders.html.DirectoryHTMLBuilder">More info on sphinx builders</a>.',
+                max_length=20,
+                verbose_name="Documentation type",
+            ),
+        ),
```
Comment (stsewd) [inline readthedocs/projects/migrations/0116_remove_old_issues.py:102]: Seems like this field is being set a non-null again here instead of being deleted.

Comment (humitos) [reply readthedocs/projects/migrations/0116_remove_old_issues.py:102]: Hrm... Not sure what happened here. I will try to re-create this migration again.

Comment (humitos) [reply readthedocs/projects/migrations/0116_remove_old_issues.py:102]: I re-ran the `makemigrations` command and it's seems that Django detects this as an `AlterField` for some reason on both models.

Comment (humitos) [reply readthedocs/projects/migrations/0116_remove_old_issues.py:102]: I found the issue. The field is duplicated. It's also present in https://github.com/readthedocs/readthedocs.org/blob/5c556ec4bc04d3566ce88cace101f330ea48ca41/readthedocs/projects/models.py#L315-L325

Comment (humitos) [reply readthedocs/projects/migrations/0116_remove_old_issues.py:102]: Actually, it seems this field is being used in other places as well. So, I will keep it for now and focus on the other ones. We can remove it later.

Comment (humitos) [reply readthedocs/projects/migrations/0116_remove_old_issues.py:102]: BTW, this was a _great_ finding! Thanks for this review 👍🏼

**Comment by humitos** [comment]
We are removing some old project data here as cleanup. Is there any reason why we wouldn't want to do this? cc @stsewd @agjohnson @ericholscher

**Comment by stsewd** [comment]
> We are removing some old project data here as cleanup. Is there any reason why we wouldn't want to do this? 

Maybe to help migrate people to use a config file, but if we no longer show those values in the UI, don't think there is a strong reason.

**Comment by humitos** [comment]
> Maybe to help migrate people to use a config file, but if we no longer show those values in the UI,

Yeah, we don't show them anymore. We showed them for a few months to help with the migration before removing them from the UI.

**Review by stsewd — APPROVED** [review]

**Comment by humitos** [comment]
I created DB snapshots just in case. I going to merge this PR.

**Pull request merged**